app/models/comment.rb

```diff
@@ -1,2 +1,12 @@
 class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+
```

Comment:
- [ ] Optional you do not need a class_name 'Post'.
- [ ] It will be nice to have Rails active callbacks added